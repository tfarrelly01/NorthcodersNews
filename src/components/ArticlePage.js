import React from 'react';

const ArticlePage = () => <h1>Article Page</h1>;

export default ArticlePage;