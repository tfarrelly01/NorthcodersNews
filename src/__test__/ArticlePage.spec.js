import React from 'react';
import { shallow } from 'enzyme';
import ArticlePage from '../components/ArticlePage';

test('ArticlePage renders correctly', () => {
  const component = shallow(<ArticlePage />);
  expect(component).toMatchSnapshot();
});