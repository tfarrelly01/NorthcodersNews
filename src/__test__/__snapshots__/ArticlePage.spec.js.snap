// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticlePage renders correctly 1`] = `
<h1>
  Article Page
</h1>
`;
